radish/documentors/__main__.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module provides a script to document Radish step implementations
+"""
+
+from sys import exit
```

Comment:
Please use `import sys` / `import os` for such modules. I think it's more confusion to just import those specific functions than in brings any benefits. It loads the entire module any ways. 

If you import it with `import sys` and use `sys.exit()` there won't be any confusion about which function we are talking about. (Indeed did you actually overwrite the built-in `exit()` function)